WordPressComKit/DateHelpers.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public func convertUTCWordPressComDate(dateString: String) -> NSDate? {
```

Comment:
It has the potential of being reused elsewhere - I dislike using `@testable` more than making this `internal` :grin: